acj/acj_algorithms/score_acj/__init__.py

```diff
@@ -0,0 +1,158 @@
+import math
+from collections import namedtuple
+
+from acj.acj_algorithms.comparison_pair import ComparisonPair
```

Comment:
NIT: can we name the package `algorithms` instead of `acj_algorithoms`. It seems a little long to me.